src/account_mt5.py

```diff
@@ -8,17 +8,18 @@ class AccountMT5(IAccount):
     balance: float
     profit: float
 
-    ticket: int # do we use this ticket variable in live implementation ?
-    symbol: object # should we pass a symbol object to this class ? 
+    ticket: int # do we use this ticket variable in live implementation ? 1.21.24 - to decide
+    symbol: object # should we pass a symbol object to this class ? 1.21.24 - implemented in sim class, need to add to live implementation
```

Comment:
I don't believe we will want a symbol in this class. We want the account the balance on trades on all symbols. If we pass symbols into the account class we are saying that we will be trading on this symbol and this symbol only.

Further, the account class should be isolated only to things that the account needs to know about so that it doesn't worry about non-account things and other classes don't worry about the things that account knows about.

Reply:
Also agree, symbols made sense to pass to the sim class to do its operations but it is not required in the account_executor_mt5, will be removed.

Reply:
See update

Reply:
See update